seqr/views/utils/orm_to_json_utils.py

```diff
@@ -867,3 +867,10 @@ def _process_result(data, model):
         data['isSignificant'] = data['pAdjust'] < model.SIGNIFICANCE_THRESHOLD
 
     return _get_json_for_models(models, process_result=_process_result, **kwargs)
+
+
+def get_json_for_phenotype_prioritization(models, **kwargs):
+    def _process_result(data, model):
+        data['scores'] = {_to_camel_case(score): value for score, value in data['scores'].items()}
```

Comment:
I think we should do this at loading time and store things in the database the way we want them, instead of post processing them here